### PR TITLE
No more jQuery

### PR DESCRIPTION
**Pull request by nickvergessen**
Signed-off-by: Joas Schilling <coding@schilljs.com>

### CODE REVIEW

File: core/js/publicshareauth.js [hunk 1]
```diff
@@ -1,9 +1,19 @@
-$(document).ready(function(){
- 	$('#password').on('keyup input change', function() {
-		if ($('#password').val().length > 0) {
-			$('#password-submit').prop('disabled', false);
-		} else {
-			$('#password-submit').prop('disabled', true);
-		}
-	});
+function ready(fn) {
+	if (document.readyState !== 'loading') {
```
Comment (skjnldsv) [inline core/js/publicshareauth.js]: why this?
Isn't domcontentloaded enough?

Comment (nickvergessen) [reply core/js/publicshareauth.js]: That is what http://youmightnotneedjquery.com/ suggested on `$(document).ready`

Comment (skjnldsv) [reply core/js/publicshareauth.js]: https://caniuse.com/#feat=domcontentloaded is compatible with is :)
This should be enough ;)

Comment (nickvergessen) [reply core/js/publicshareauth.js]: Fixed

**Review by skjnldsv — APPROVED** [review]

**Review by juliusknorr — APPROVED** [review]

**Review by ChristophWurst — APPROVED** [review]

**Pull request merged**